README.md

```diff
@@ -58,8 +71,32 @@ A commented template config will be written as part of the `eris chains make` [p
 
 ## Contribute
 
-See the [CONTRIBUTING.md](.github/CONTRIBUTING.md) for more details.
+We welcome all contributions and have submitted the code base to the Hyperledger project governance during incubation phase.  As an integral part of this effort we want to invite new contributors, not just to maintain but also to steer the future direction of the code in an active and open process.
+
+You can find us on:
+- [the Marmot Den (slack)](http://slack.monax.io)
+- [here on Github](http://github.com/eris-ltd/eris-db/issues)
+- [support.monax.io](http://support.monax.io)
+
 
 ## License
 
 [Apache 2.0](license.md)
+
+## Future work
+
+Some burrows marmots have already started digging to build the enterprise ecosystem applications of the future are listed below.  Marmots live in groups we welcome your help on these or other improvement proposals.
+
+#### Security, Identity and Privacy
+
+- **(Q2) Advanced security framework:** Functionality preceding Monax’s Nightsky (see multi-chain universe continuation below) to enable global encryption of transactions, genesis files and account storage at rest.
+- **(Q2) Identity Management and Whitelisting:** write protection is inherent to the permissioning scheme but read protection will be enabled by integrating certificates for restricting connectivity access to the peer-to-peer network for validators, verifiers or light clients.  This is in addition to normal VPN and firewall solutions and a precursor of Monax’s Nightsky, which restricts read-access through selective decryption.
+- **(Q3) Multi-chain Universe (Step 2 of 3):** The second step towards enabling the multi-chain universe is well-underway with the Tendermint Cosmos proposal for interblockchain communication.  
+- **(Q3) Multi-chain Universe (Step 3 of 3):** The third and final step is the Monax’s Nightsky proposal for inherent encryption of transactions and allowing only partial reconstruction of the smart contract application state for verifiers or light clients (for mobile or IoT devices). Together Cosmos and Nightsky allow smart contracts to orchestrate where information flows and who has the ability to decypher what part of the data.
```

Comment:
It's a rename of the internal Skywalker proposal; I preferred avoiding copyright infringements :)